dramsim3.cfg.ini

```diff
@@ -0,0 +1,67 @@
+[dram_structure]
```

Comment:
put this file in the ipemu/dramsim folder

Reply:
If we don't run dramsim on CI, we can delete this file all together